### PR TITLE
Update gwb to v1.0.0

### PR DESCRIPTION
**Pull request by MFraters**
A new version (version 1.0.0!) of the world builder will be released soon (See https://github.com/GeodynamicWorldBuilder/WorldBuilder/issues/746 for todo's) which is related to the world builder JOSS paper, so it is also time to start the review process to get it into aspect. This pull request will still be update until the final release.

### CODE REVIEW

File: contrib/world_builder/CITATION [hunk 1]
```diff
@@ -10,14 +22,24 @@ URL = {https://se.copernicus.org/articles/10/1785/2019/},
 DOI = {10.5194/se-10-1785-2019}
 }
 
-@misc{gwb-doi-v0.4.0,
-  title        = {{The Geodynamic World Builder} v0.4.0 [software]},
+@misc{gwb-doi-v1.0.0,
+  title        = {{The Geodynamic World Builder} v0.6.0 [software]},
```
Comment (gassmoeller) [inline contrib/world_builder/CITATION:26]: ?
```suggestion
  title        = {{The Geodynamic World Builder} v1.0.0 [software]},
```

File: contrib/world_builder/CMakeLists.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.13.4)
+cmake_minimum_required(VERSION 2.8.12)
```
Comment (gassmoeller) [inline contrib/world_builder/CMakeLists.txt:1]: I think we had adjusted this to 3.13.4 because that is the minimum version for ASPECT

**Review by gassmoeller — COMMENTED** [review]
I mostly looked at the files that are relevant for the ASPECT integration and they look good. Just two small comments.

**Comment by gassmoeller** [comment]
But you will have to look into the failing tests, it looks as if your version check might not be working correctly.

**Comment by gassmoeller** [comment]
And the documentation tester failure should be fixed on a recent master. Could you rebase your PR to confirm?

**Comment by MFraters** [comment]
@gassmoeller, thanks for the review! The documentation tester is now indeed fixed and I have addressed your other comments. I think the other tests will be fixed as well, but we will have to see :)

**Comment by MFraters** [comment]
GWB v1.0.0 has been release (see https://github.com/GeodynamicWorldBuilder/WorldBuilder/releases/tag/v1.0.0), so this should be ready to merge :)

**Review by gassmoeller — APPROVED** [review]

**Pull request merged**